tests/conftest.py

```diff
@@ -50,3 +51,134 @@ def templates(read_config, get_hooks) -> list:
             "hooks": get_hooks(path),
         }
     return templates
+
+
+@pytest.fixture(scope="session")
+def cookieplone_root() -> dict:
+    """Cookieplone root dir."""
+    folder = Path(__file__).parent
+    return folder.parent.resolve()
+
+
+@pytest.fixture(scope="module")
+def cookies_module(request, tmpdir_factory, _cookiecutter_config_file):
+    """Yield an instance of the Cookies helper class that can be used to
+    generate a project from a template.
+
+    Run cookiecutter:
+        result = cookies.bake(extra_context={
+            'variable1': 'value1',
+            'variable2': 'value2',
+        })
+    """
+    template_dir = request.config.option.template
+
+    output_dir = tmpdir_factory.mktemp(f"cookies-{request.module.__name__}")
+    output_factory = output_dir.mkdir
+
+    yield Cookies(template_dir, output_factory, _cookiecutter_config_file)
+
+    # Add option to keep generated output directories.
+    if not request.config.option.keep_baked_projects:
+        output_dir.remove()
+
+
+@pytest.fixture
+def volto_versions():
+    versions = ["17.15.5", "16.31.4", "18.0.0-alpha.27"]
+    return versions
+
+
+@pytest.fixture
+def plone_versions():
+    versions = [
+        "5.0rc1",
+        "5.0rc2",
+        "5.0rc3",
+        "5.1.0",
+        "5.1.1",
+        "5.1.2",
+        "5.1.3",
+        "5.1.4",
+        "5.1.5",
+        "5.1.6",
+        "5.1.7",
+        "5.1a1",
+        "5.1a2",
+        "5.1b1",
+        "5.1b2",
+        "5.1b3",
+        "5.1b4",
+        "5.1rc1",
+        "5.1rc2",
+        "5.2.0",
+        "5.2.1",
+        "5.2.10",
+        "5.2.11",
+        "5.2.12",
+        "5.2.13",
+        "5.2.14",
+        "5.2.2",
+        "5.2.3",
+        "5.2.4",
+        "5.2.5",
+        "5.2.6",
+        "5.2.7",
+        "5.2.8",
+        "5.2.9",
+        "5.2a1",
+        "5.2a2",
+        "5.2b1",
+        "5.2rc1",
+        "5.2rc2",
+        "5.2rc3",
+        "5.2rc4",
+        "5.2rc5",
```

Comment:
Remove versions out of support?

```suggestion
```

Reply:
@stevepiercy This is only meant to be used with tests, I won't mind keeping this outdated, but I will merge both suggestions